buildpacks/ruby/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- buildpacks/ruby: Introduce heroku build metrics support (https://github.com/heroku/buildpacks-ruby/pull/172)
```

Comment:
This changelog is specific to `buildpacks/ruby` - why is the `buildpacks/ruby` prefix needed here (there's nothing for it to differentiate against)?